### PR TITLE
Minzoom fix

### PR DESCRIPTION
**Pull request by lowriech**

### CODE REVIEW

**Comment by gislawill** [comment]
Heads up, you can pull in the latest from [feature/cl-updates-late-jan](https://github.com/CoastalResilienceExplorer/GlobalFloodExplorer/tree/feature/cl-updates-late-jan) to fix the cypress tests here

File: src/panels/stats-panel/stats-panel-container.js [hunk 1]
```diff
@@ -163,22 +164,45 @@ function OpenToggle({ isOpen, setIsOpen }) {
...
+    <div
+      className={
+        "open-toggle-container" +
+        (selectedFeatures.length !== 0 && !isOpen ? " coral" : "") +
+        (selectedFeatures.length === 0 ? " disabled" : "")
+      }
+      onClick={() => {
+        selectedFeatures.length > 0 && setIsOpen(!isOpen);
+      }}
+    >
```
Comment (gislawill) [inline src/panels/stats-panel/stats-panel-container.js:178]: Let's use a <button> here instead, it got better support for screen readers (for those poor of sight) than divs with onClicks and it won't require any cursor style handling in our css

Comment (lowriech) [reply src/panels/stats-panel/stats-panel-container.js:178]: Not sure I know what you mean.

**Comment by lowriech** [comment]
@gislawill Still getting the cypress failure here after pulling in your `data-test-id` change

**Pull request closed without merging**